examples/python/agent-samples/youtube-shorts-assistant/agent.py

```diff
@@ -0,0 +1,52 @@
+# Shows how to call all the sub-agents using the LLM's reasoning ability. Run this with "adk run" or "adk web"
+
+from google.adk.agents import LlmAgent
+from google.adk.tools import google_search
+
+from util import load_instruction_from_file
```

Comment:
Loading errors and may we can further simplify this.
```suggestion
```

Reply:
ditto